### PR TITLE
Finalize javascript_nodejs/04.simple-prompts

### PR DESCRIPTION
**Pull request by benbrown**
- [x] Is numbered and named correctly and following the spec
- [x] All files have been spell checked
- [x] All style cop/ linting issues have been addressed
- [x] Include all DCR changes
- [x] Explicit state save
- [x] Method name changes begin, end etc
- [x] Does not use mainDialog unless that file actually extends Dialog/ ComponentDialog
- [x] Compiles with latest packages
- [x] Runs per instructions in README
- [x] Add commented out code to use blob storage
- [x] ~Update to look for production/ local (bot configuration)~
- [x] onTurnError at adapter
- [x] Include .env or appsettings.json or web.config with botFilePath and botFileSecret
- [x] Deployment scripts folder has been cleaned up and only includes msbot export files
- [x] README to include
   - [x] ~msbot clone steps~
   - [x] ~Tools install/ uninstall~
   - [x] Concise set of steps to run the sample
- [x] ~C# sample -- Include .bot file in the build output~
- [x] All samples include the latest botfile no "secretKey" field; replaced with "padLock"
- [x] remove and sort usings
- [x] ~remove azuredeploy.* under deploymentscripts folder (if any)~

### CODE REVIEW

**Comment by benbrown** [comment]
I'm not sure if I need to include deployment scripts with this one. If a bot has no external services, do we want to include that stuff?  @vishwacsena @sgellock

File: samples/javascript_nodejs/04.simple-prompt/README.md [hunk 1]
```diff
@@ -5,9 +5,9 @@ This sample shows how to use the prompts classes included in `botbuilder-dialogs
     ```bash
     git clone https://github.com/microsoft/botbuilder-samples.git
     ```
-- In a terminal, navigate to javascript_nodejs/04.simple-prompt-bot
+- In a terminal, navigate to samples/javascript_nodejs/04.simple-prompt
     ```bash
-    cd javascript_nodejs/04.simple-prompt-bot
+    cd samples/javascript_nodejs/04.simple-prompt
     ```
     - Point to the MyGet feed
```
Comment (sgellock) [inline samples/javascript_nodejs/04.simple-prompt/README.md]: we shouldn't have any references and instructors to myGet feeds.

File: samples/javascript_nodejs/04.simple-prompt/bot.js [hunk 2]
```diff
@@ -90,11 +90,11 @@ class MainDialog {
 
             // Show menu if no response sent
```
Comment (vishwacsena) [inline samples/javascript_nodejs/04.simple-prompt/bot.js]: Unrelated to this PR, can you require { ActivityTypes } from 'botbuilder' and use ActivityTypes.Message in line 72?

File: samples/javascript_nodejs/04.simple-prompt/.gitignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+node_modules/
```
Comment (vishwacsena) [inline samples/javascript_nodejs/04.simple-prompt/.gitignore]: Unrelated to this PR, let's include botFileSecret=
in the .env file.

File: samples/javascript_nodejs/04.simple-prompt/bot.js (hunk 2 above)
Comment (benbrown) [reply samples/javascript_nodejs/04.simple-prompt/bot.js]: done

**Pull request merged**